### PR TITLE
Show empty state when there are no documents available

### PR DESCRIPTION
**Pull request by dephraiim**
## Screenshot Before
Papermark Ephraim Atta-Duncan

## Screenshots After
Papermark Ephraim Atta-Duncan PR Screenshot
Papermark Ephraim Atta-Duncan

### CODE REVIEW

File: components/Sidebar.tsx
Comment (mfts) [inline components/Sidebar.tsx:1]: By removing the "New Link" button from the Sidebar component, it won't be available anywhere. Make sure it's only removed on the /documents page, but still visible on other pages.

**Review by mfts — CHANGES_REQUESTED** [review]

File: components/Sidebar.tsx [hunk 1]
```diff
@@ -178,16 +180,19 @@ export default function Sidebar({ children }: { children: React.ReactNode }) {
...
-              <Link
-                href="/documents/new"
-                type="button"
-                className="relative inline-flex items-center gap-x-1.5 rounded-md bg-indigo-500 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-indigo-400 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-500"
-              >
-                <PlusIcon className="-ml-0.5 h-5 w-5" aria-hidden="true" />
-                New Link
-              </Link>
-            </div>
+            {!isDocumentsRoute ? (
```
Comment (mfts) [inline components/Sidebar.tsx]: You need to add a check in to show the "new link" button if there are documents available. 

Right now it only checks for the route and assumes `documents.length == 0`

**Review by mfts — CHANGES_REQUESTED** [review]

**Comment by mfts** [comment]
Thanks @dephraiim. I added some small changes

**Comment by dephraiim** [comment]
Looks like I missed the changes request, sorry about that.

**Pull request merged**